iopipe/config.py

```diff
@@ -12,7 +12,7 @@ def set_config(**config):
     config.setdefault('enabled', bool(strtobool(os.getenv('IOPIPE_ENABLED', 'true'))))
     config.setdefault('host', get_hostname())
     config.setdefault('install_method', 'manual')
-    config.setdefault('network_timeout', 5000)
+    config.setdefault('network_timeout', 5)
```

Comment:
Double checking: this change is because this value is actually seconds, not milliseconds? (5000 would be assuming ms, while 5 must be sec?). Good fix.

Reply:
Yes, requests accepts seconds and not milliseconds. It was set to `5000` originally because that's what the node.js agent was defaulting to, but it doesn't make sense to do `int(milliseconds / 1000)` everywhere. The timeout handler PR has a similar situation, but we may actually want millisecond granularity there.

Reply:
👍